docs/Overview.md

```diff
@@ -49,15 +55,18 @@ The technical model describing the monitoring requirements for a receiver is [Nc
 
 This model MUST inherit from the baseline status monitoring model [NcStatusMonitor](https://specs.amwa.tv/nmos-control-feature-sets/branches/publish-status-reporting/monitoring/#ncstatusmonitor)
 
-The purpose of the overall status is to abstract and combine the specific domain statuses of a monitor into a single status which can be more easily observed and displayed by a simple client. A good practice is to populate the status message property with details of the worst status causing the current value of the overall status.
+The purpose of the overallStatus is to abstract and combine the specific domain statuses of a monitor into a single status which can be more easily observed and displayed by a simple client. A good practice is to populate the status message property with details of the worst status causing the current value of the overallStatus.
```

Comment:
Slightly contradictory non-conformance/conformance language:

_"A **good practice** is to populate the status message property with details of the worst status causing the current value of the overallStatus"_
vs. 
_"Devices **MUST** follow the rules listed below when mapping specific domain statuses in the combined overallStatus:"_ below.  

Perhaps remove the "good practice" sentence from here to avoid confusion?